core/src/i18n/pl.yaml

```diff
@@ -59,8 +56,6 @@ cron-jobs:
   title: Cron Jobs
 custom-resource-definitions:
   title: Definicje zasobów niestandardowych
-deployments:
-  title: Deploymenty
 discovery-and-network:
   title: Odkrywanie i Sieć
 functions:
```

Comment:
Re line 95:
It seems to me that the word order is wrong. It should be 
```
      error: 'Błąd podczas ładowania danych {{title}} z powodu: {{message}}'
```

For example: `Błąd podczas ładowania danych Service Catalog`, and not `błąd ładowania Service Catalog danych`. 
Also, consider adding a `dla` before the title (`danych dla Service Catalog`).